tests/specs/infinite-gas.k

```diff
@@ -9,18 +10,18 @@ requires "evm.md"
  // In particular, this means that `#gas(_) <Int #gas(_) => false`, and `#gas(_) <=Int #gas(_) => true`, regardless of the values contained in the `#gas(_)`.
  //
 
-module INFINITE-GAS
-    imports INFINITE-GAS-JAVA
-    imports INFINITE-GAS-HASKELL
+module INFINITE-GAS-LEMMAS
+    imports INFINITE-GAS-LEMMAS-JAVA
+    imports INFINITE-GAS-LEMMAS-HASKELL
 endmodule
 
-module INFINITE-GAS-HASKELL [kore]
-    imports INFINITE-GAS-COMMON
+module INFINITE-GAS-LEMMAS-HASKELL [kore]
+    imports INFINITE-GAS-LEMMAS-COMMON
 
-    rule #gas(_)  <Int _C => false [concrete(_C), simplification]
-    rule #gas(_) <=Int _C => false [concrete(_C), simplification]
-    rule #gas(_) >=Int _C => true  [concrete(_C), simplification]
-    rule _C <=Int #gas(_) => true  [concrete(_C), simplification]
+    syntax Int ::= #gas ( Int , Int , Int ) // startGas, nonMemory, memory
+ //-----------------------------------------------------------------------------------
+
```

Comment:
Leave this declaration in `INFINITE-GAS-COMMON`, and remove other declarations of this one in their respective test-suites (eg. `tests/specs/erc20/abstract-semantics-segmented-gas.k`).